### DISCUSSION TITLE
`to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead？

### DISCUSSION THREAD

**Discussion by AustinYuAo** [Project-MONAI/MONAI#3894] [Q&A]
Hi，I tried to use AsDiscrete(to_onehot=True,n_classes=args.out_channels) ，but always go wrong `to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead. 
My monai version is 0.8.0，how can I handle it？

**Comment by Nic-Ma** [comment]
Hi @aubest ,

Thanks for your interest here.
As the warning said, please update to `to_onehot=args.out_channels` and try again.

Thanks.
